packages/esm-patient-common-lib/src/workspaces/useWorkspaces.tsx

```diff
@@ -1,22 +1,25 @@
 import { useEffect, useMemo, useState } from 'react';
 import { getWorkspaceStore, OpenWorkspace, WorkspaceWindowState } from '@openmrs/esm-patient-common-lib';
-import { Prompt, WorkspaceStoreState } from './workspaces';
+import { Prompt, WorkspaceStoreState, updateWorkspaceSizeState } from './workspaces';
 
 export interface WorkspacesInfo {
   active: boolean;
-  windowState: WorkspaceWindowState;
+  workspaceWindowState: WorkspaceWindowState;
   workspaces: Array<OpenWorkspace>;
   prompt: Prompt;
+  updateWorkspaceSizeState: (val: WorkspaceWindowState) => void;
```

Comment:
Can we pick either `size` or `state` and just be consistent about it? I feel like either workspaceSize or workspaceState is fine.

Reply:
Updated to `updateWorkspaceWindowState`, matches with it's value `workspaceWindowState`